repositories/variable.go

```diff
@@ -31,4 +31,7 @@ type Variable struct {
 
 	// Value defines the value of the variable.
 	Value string `json:"value" mapstructure:"value" yaml:"value"`
+
+	// Pattern defines a regex pattern for valid values.
+	Pattern string `json:"pattern" mapstructure:"pattern" yaml:"pattern"`
```

Comment:
by defining it this way, does it allow someone to define a `.pathogen.yml` in json or yaml?

Reply:
Technically the structure can be deserialized from JSON, YAML or from a `map[string]interface{}` as defined.  However the implementation right now requires YAML (https://github.com/galaho/pathogen/blob/7682850cb9ef8938b6d6b7d1c0e307608aca9952/repositories/repository.go#L145).  It could easily be changed though to try YAML then JSON.  Just don't see a need right now, do you?

Reply:
No. Just trying to understand the struct.